test/l.jl

```diff
@@ -237,14 +239,14 @@
         @test lstled(11.1, array) == 6
     end
 
-    # ERROR: doesnt pass its test
-    # let array = [-2, -2, 0, 1, 1, 11]
-    #     @test lstlei(-3, array) == 0
-    #     @test lstlei(-2, array) == 2
-    #     @test lstlei(0, array) == 3
-    #     @test lstlei(1, array) == 5
-    #     @test lstlei(12, array) == 6
-    # end
+    # ERROR: some don't pass their test
```

Comment:
This should return the index of the biggest integer which is smaller or equal to the given integer, but it gives an index which is one higher. The same happens with the other erroring function